src/sentry/models/options/project_option.py

```diff
@@ -152,6 +152,9 @@ def post_save(self, instance: ProjectOption, **kwargs: Any) -> None:
     def post_delete(self, instance: ProjectOption, **kwargs: Any) -> None:
         self.reload_cache(instance.project_id, "projectoption.post_delete")
 
+    def isset(self, project: Project, key: str) -> bool:
```

Comment:
could you make this `is_set`? or is isset something else?

Reply:
also TIL about `Ellipsis`

Reply:
for `isset` it seems like there's already a pattern established: https://github.com/search?q=repo:getsentry/sentry%20isset&type=code - specifically around project options too. #duckTypingAndAllThat

Reply:
weird, I don't like it but ok!